modulus/utils/zenith_angle.py

```diff
@@ -74,10 +67,45 @@ def cos_zenith_angle(
     """
     lon_rad = np.deg2rad(lon, dtype=dtype)
     lat_rad = np.deg2rad(lat, dtype=dtype)
-    return _star_cos_zenith(time, lon_rad, lat_rad)
+    julian_centuries = _days_from_2000(time) / 36525.0
+    return _star_cos_zenith(julian_centuries, lon_rad, lat_rad)
+
+
+def cos_zenith_angle_from_timestamp(
+    timestamp: T,
+    lon: T,
+    lat: T,
+) -> T:
+    """
+    Compute cosine of zenith angle using UNIX timestamp
+
+    Since the UNIX timestamp is a floating point or integer this routine can be
+    compiled with jax.
 
+    Parameters
+    ----------
+    timestamp: timestamp in seconds from UNIX epoch
+    lon: longitude in degrees E
+    lat: latitude in degrees N
 
-def _days_from_2000(model_time):  # pragma: no cover
+    Example:
+    --------
+    >>> model_time = datetime.datetime(2002, 1, 1, 12, 0, 0)
+    >>> angle = cos_zenith_angle(model_time.timestamp(), lat=360, lon=120)
+    >>> abs(angle - -0.447817277) < 1e-6
+    True
+    """
+    lon_rad = np.deg2rad(lon, dtype=dtype)
+    lat_rad = np.deg2rad(lat, dtype=dtype)
+    seconds_in_day = 86400
+    days_in_julian_year = 36525.0
```

Comment:
Why is this not `365.25` or the variable `days_in_julian_century`?

Reply:
fixed.